package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1080,13 +1082,18 @@ def get_attr(attrname, default):
         altlocs = get_attr('altLocs', ' ')
         resnames = get_attr('resnames', 'UNK')
         icodes = get_attr('icodes', ' ')
+        chainids = get_attr('chainIDs', ' ')
```

Comment:
How will you distinguish between not having a chainID (``not hasattr(obj, "chainID")`` and having an empty chainID?